lib/dartservices_client/v1.dart

```diff
@@ -1,3 +1,5 @@
+// This is a generated file (see the discoveryapis_generator project).
```

Comment:
This is already a pretty big PR; let's land the generated discovery API in another PR (once it lands in the dart-services project), and update this PR from master.